src/include/utime.h

```diff
@@ -400,17 +381,45 @@ class utime_t {
 
     const char *p = strptime(date.c_str(), "%Y-%m-%d", &tm);
     if (p) {
-      if (*p == ' ') {
+      if (*p == ' ' || *p == 'T') {
```

Comment:
rgw multisite is using utime_t::gmtime_nsec() to send timestamps between zones in json format. it looks like older zones will have trouble parsing the timestamps from octopus zones as a result. can we plan on backporting these utime_t::parse_date() changes to prepare for this mixed cluster case?

Reply:
Yeah.  We'll need to document the minimum mimic and nautilus versions to run before doing the upgrade to octopus.

Reply:
i'll work on backports under http://tracker.ceph.com/issues/40086